### PR TITLE
New Homepage second pass

### PR DESCRIPTION
**Pull request by johnnyd-eth**
Another pass of JB-96

As before, I think it would be best to just make **code notes** for me to fix in upcoming PRs so we can now get this one to Strath to make some comments on the UI.

- Main changes are mobile-related, it's all tight I believe
- A bunch of other small updates I noticed were missing from the first pass

**Here's what's still remaining:**
*Blocking:*
- Horizontal scroll arrows missing for project carousals on desktop
- Juicy picks DESKTOP incomplete
- Remove new Ant-d code

*Non-blocking?:*
- Everything related to project tags
- New ETH icons (will need to be a solid site-wide change and require its own pr)
- Icons next to list in 'How Juicebox works' missing
- FAQ style update

### CODE REVIEW

File: src/pages/home/SuccessStoriesSection/SuccessStoriesSection.tsx [hunk 1]
```diff
@@ -21,32 +20,31 @@ export function SuccessStoriesSection() {
...
       />
-      <Row gutter={DEFAULT_HOMEPAGE_GUTTER}>
-        {topProjects?.map(p => (
-          <SuccessStoriesCard project={p} key={p.metadataUri} />
-        ))}
-      </Row>
-      <div className="text-center">
-        <Space direction="vertical" className="mt-16 w-full" size="large">
-          {/* SOON WILL HAVE A LINK TO "CASE STUDIES" HERE */}
+      <ProjectCarousel
```
Comment (tomquirk) [inline src/pages/home/SuccessStoriesSection/SuccessStoriesSection.tsx:27]: awesome call, nice one

File: src/pages/home/JuicyPicksSection/JuicyPicksSection.tsx [hunk 2]
```diff
@@ -17,9 +15,7 @@ export function JuicyPicksSection() {
           <Trans>Peep our selection of top trending projects this month.</Trans>
         }
       />
-      <Row gutter={DEFAULT_HOMEPAGE_GUTTER}>
-        {isMobile ? <JuicyPicksMobile /> : <JuicyPicks />}
-      </Row>
+      {isMobile ? <JuicyPicksMobile /> : <JuicyPicks />}
```
Comment (tomquirk) [inline src/pages/home/JuicyPicksSection/JuicyPicksSection.tsx:18]: How different is mobile to desktop? Do we really need 2 components? is there a solution where one component will work?

File: src/pages/home/HomepageProjectCard.tsx [hunk 3]
```diff
@@ -46,33 +46,38 @@ export function HomepageProjectCard({
...
+      <a
+        className={`h-full w-[280px] cursor-pointer overflow-hidden rounded-lg px-[10px]`}
+        style={{ flex: '0 0 auto' }}
+      >
+        <div
+          className={`h-full rounded-lg ${PROJECT_CARD_BORDER} ${PROJECT_CARD_BG}`}
+        >
           {projectLogo}
-          <div className="rounded-lg p-5">
+          <div className="flex flex-col justify-between gap-4 rounded-lg p-5">
```
Comment (tomquirk) [inline src/pages/home/HomepageProjectCard.tsx:57]: Nice!

**Review by tomquirk — CHANGES_REQUESTED** [review]
@johnnyd-eth lgtm! nice work. One question about different mobile/desktop views

File: src/pages/home/JuicyPicksSection/JuicyPicksSection.tsx (hunk 2 above)
Comment (johnnyd-eth) [reply src/pages/home/JuicyPicksSection/JuicyPicksSection.tsx:18]: Definitely need two components I think. Mobile is just normal slider but desktop is this totally different layout: 

Screen Shot 2023-04-13 at 7 14 34 am

Screen Shot 2023-04-13 at 7 14 40 am

Comment (tomquirk) [reply src/pages/home/JuicyPicksSection/JuicyPicksSection.tsx:18]: I think we can merge these 2 components:
- on mobile, hide the left col
- on mobile, nowrap left col, horizontal scroll

**Pull request merged**